instructional-design/readme.md

```diff
@@ -0,0 +1,151 @@
+
+## 💡Key Tips
+
+1. 🎯Start with the motivation: why is this useful to me?
+2. ✅Give an example solution: what does success look like?
+3. 🍱Create an analysing, categorising, summarising, illustrating, or explaining activity
+4. 📌Set the initial task (3 and 4 can be in either order)
+5. 🧪Make it testable. How will the _learner_ assess their understanding?
+
+```mermaid
+flowchart LR
+A[📍 Learner] <-->|🕵🏾 connect| B[Teacher]
+B -->|👣 extend| C([📍 Concept])
+```
+
+---
+
+## ✏️Worksheet:
+
+## Write a Personal Statement Task Revision
+
+<div contenteditable=true>
+Work on your revision on a document or a piece of paper. 
+</div>
+
+---
+
+## ✅ Worksheet worked example
+
+## Personal statement
+
+### 🧩 What is the FORMAT?
+
+- 1 paragraph
+- No personal pronouns
+- 50 to 150 words
+- At the top of a CV straight after your name and contact information
+
+Career changer with ten years solid experience in original field, but latterly caught the coding bug. Spent the last two years working on software development obsessively. Build real products as a volunteer in small teams every week. Building with TypeScript, React, Node, Deno, Rust, GraphQL - just name it. Automating the deployment of containerised applications to AWS just for fun. Linked portfolio is stuffed with original, inventive, useful passion projects. Excited to learn more. More more more!
+
+### 🎯 What are the GOALS?
+
+A great personal statement grabs you from the first line. The point is to make you, particularly you, stand out from the hundreds of people your reader is sifting through. This is your best chance to clearly connect your specific personal achievements with some of the skills listed on the job description in a short, clear, paragraph. Your reader comes away thinking "I want to interview _this_ one!" An effective personal statement is confident, distinctive, and shares just enough to make you want to **read the rest of the CV**.
+
+### 🧪 And how do we TEST?
+
+#### Make a game
+
+_My suggestions: We are building confidence and expression here. We don't need to grade these attempts, we need to stage opportunities for the learner to evaluate effective and ineffective texts, including but not limited to their own._
+
+##### 🪡 Energiser: Haystack
+
+Take the names off everyone's personal statement and mix them up (and add in some AI generated ones). Now try to match the statement to the person. The person who is the most easily identifiable wins.
+
+##### ⏲️ Energiser: Play recruiter
+
+Give everyone a job description and a big stack of CVs. Set a timer for 5 minutes. You must shortlist to three candidates. If your candidates are the "best match", you win!
+
+##### 💡 Energiser: Buzz off!
+
+(Sorting words is a familiar strategy for identifying categories. Categorising is a key learning activity. It helps us organise and structure our understanding.)
+
+1. First exercise: sort the words into two groups: expressive and obscuring / straightforward and jargon etc
```

Comment:
The exercise says to identify two groups, but there's just one (unlabelled) group below? I would at least label the group, and maybe add a second?